src/main/java/com/laserfiche/repository/api/clients/impl/model/FindEntryResult.java

```diff
@@ -0,0 +1,82 @@
+package com.laserfiche.repository.api.clients.impl.model;
```

Comment:
Ideally, let's also format this file (ctrl+alt+shift+l) for consistency's sake.

Reply:
The issue mentioned in the comment is still there.

Reply:
> See comments.

ok so for all the formatting comments, after importing that xml file i did ctrl + alt + shift + L -> whole file and optimize imports for all the files